models/app.model.js

```diff
@@ -25,6 +25,22 @@ exports.selectArticles = (article_id) => {
   });
 };
 
+exports.selectArticleComments = (article_id) => {
+  let queryString = `SELECT * FROM comments `;
+  const queryValues = [];
+
+  if (article_id) {
+    queryValues.push(article_id);
+    queryString += `WHERE article_id = $1 ORDER BY created_at DESC`;
+  }
+
+
```

Comment:
I don't think that there is a use case for `SELECT * FROM comments` on it's own so the string concatenation isn't really needed here.